### DISCUSSION TITLE
Proxy dataSource uri to local json file

### DISCUSSION THREAD

**Discussion by dburkhart** [ui5-community/ui5-ecosystem-showcase#523] [Q&A]
Hi,

I try to understand the UI5 Tooling a bit better and therefore I try to implement a UI5 application.
My thoughts: I  can use the dataSource section of the manifest.json to specify my root model, including the intended backend uri.
Then using the ui5.yaml file to configure a mapping to a local json file in order to abstract the real backend connections from its dev targets.

Is this a correct assumption?
If yes, I run into an issue regarding the ui5-middleware-simpleproxy, as my baseURI always getting an tailing slash and this will not work, as I tried to point to the resulting json file, not to a folder

I assume I have understood something totally wrong 😬

![proxy_to_local_json](https://user-images.githubusercontent.com/47185404/118650954-49e3ed00-b7e5-11eb-90d5-6ef5d85ba6d2.png)

I would really appreciate your help and hope you can give me an hint how to solve this.

Regards,
Daniel

**Comment by petermuessig** [comment]
Sorry for answering that late - ~2 years passed - and I assume you understood the middleware in the meantime. 

The `baseUrl` is not considered to be a resource which can be loaded with the `mountPath` as an alias. If this is necessary it middleware would require to add a proper slash handling to differ between paths and resources. If you need such a feature we can think of opening an issue for this and creating a PR.

I would close the discussion for now - if interested please create a small issue for that or feel free to contribute... 😄
